### DISCUSSION TITLE
Can I  make static components with `template_engine: serbea`?

### DISCUSSION THREAD

**Discussion by firedev** [bridgetownrb/bridgetown#808] [Q&A] [answered]
I have tried to make a static `_components/icon.liquid`, `_components/icon.serb` file. 

It's not being rendered. I try to add `{% render "icon" %}` to my `index.md` I see an error 

```
No such file or directory: src/_partials/_icon.serb`. 
```

Is it even possible?

**Comment by ayushn21** [comment, marked as answer]
If your template engine is set to serbea, then Bridgetown will look for `serb` files only. Why do you have a liquid file in there too?

As per the convention for ERB files, which Serbea also uses, all partials much be prefixed with an `_`. If you renamed your file to `_components/_icon.serb`, it should work I think. Please remember to delete the liquid file as well.

Also components are ideally for Ruby components only so I'd also recommend moving the file to `_partials/_icon.serb`.

**Reply by firedev** [reply]
Thank you, guess you right, static component IS a partial. Maybe "static components" should be removed from the docs then. Use partials or real components.
